Sources/HummingbirdXCT/HBXCTLive.swift

```diff
@@ -76,7 +43,7 @@ final class HBXCTLive<App: HBApplicationProtocol>: HBXCTApplication {
     }
 
     init(app: App) {
-        self.timeout = .seconds(15)
+        self.timeout = .seconds(20)
```

Comment:
Any particular reason for this change?

Reply:
I had a test that had an AWS request in it which was timing out. I never got the AWS timeout because the client timed out. I should probably make this configurable.

Reply:
Add Issue #318